Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -166,6 +166,7 @@ class OWTestLearners(OWWidget):
 
     class Error(OWWidget.Error):
         class_required = Msg("Train data input requires a class variable")
+        too_many_classes = Msg("Too many target variables.")
```

Comment:
I do too. And for exactly the reasons you mentioned, I asked if she can correct the other two existing messages to be more like hers.
We all want the same thing :)